deploy/charts/cert-manager/values.yaml

```diff
@@ -747,3 +747,20 @@ startupapicheck:
   # injected into pod's environment variables, matching the syntax of Docker
   # links.
   enableServiceLinks: false
+
+# Create a dynamic manifests via values:
+extraObjects: []
+  # - apiVersion: cert-manager.io/v1
+  #   kind: ClusterIssuer
+  #   metadata:
+  #     name: letsencrypt-prod
+  #   spec:
+  #     acme:
+  #       email: foo@bar.com
+  #       server: https://acme-v02.api.letsencrypt.org/directory
+  #       privateKeySecretRef:
+  #         name: letsencrypt-prod
+  #       solvers:
+  #         - http01:
+  #             ingress:
+  #               class: nginx
```

Comment:
Have you tested this? I think the `toYaml` function will choke on what is already yaml. Same as:
 * https://github.com/prometheus-community/helm-charts/issues/3328

I also do not think it will be possible / reliable to create cert-manager Custom Resources as part of the chart,
because the CRDs may not yet have been installed (possibly Helm orders those first?)
and because the cert-manager webhook takes some time to become Ready and it is not safe to create cert-manager custom resources without the webhook running.

Please supply some console output as evidence that this example works.

Reply:
Don't us a cert-manager resource in the example, because we know that won't work.
Change the example to be ConfigMap and include a template variable  in the example.
Perhaps the standard labels, like @katsew  did in https://github.com/prometheus-community/helm-charts/pull/2140
 * https://github.com/prometheus-community/helm-charts/pull/2140

And add some example `helm template` commands and output to the PR description showing how you've tested it.

Reply:
Move the example up above the default value, so that the example is included in the documentation of the values in the chart README file.
For example:

https://github.com/cert-manager/cert-manager/blob/b61de55abda95a4c273be0c8d3e6025fe8511573/deploy/charts/cert-manager/values.yaml#L7-L13

https://github.com/cert-manager/cert-manager/blob/b61de55abda95a4c273be0c8d3e6025fe8511573/deploy/charts/cert-manager/README.template.md#L76-L89